bin/drop_filter_results.py

```diff
@@ -31,8 +31,12 @@ def annotate_with_hgnc(df_family_aberrant_expression_top_hits: DataFrame, out_dr
     return df_genes.merge(df_family_aberrant_expression_top_hits, left_on="gene_id", right_on="geneID")
 
 
-def filter_by_gene_panel(df_family_top_hits: DataFrame, gene_panel: str, module_name: str) -> DataFrame:
+def filter_by_gene_panel(
+    df_family_top_hits: DataFrame, gene_panel: str, module_name: str, case_id: str, output_file_subfix: str
+) -> DataFrame:
     """Filter out from results any gene that is not present in the provided gene panel."""
+    if case_id != "":
+        case_id_ = f"{case_id}_"
```

Comment:
```suggestion
        case_id += "_"
```

Reply:
Not that fond of a variable name that is almost the same as another one aside from a trailing `_`. Alternatively that we rename it to `prefix`.